src/components/Time/Time.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import style from './Time.module.css';
+
+function getMonthName(month) {
+  const monthNames = [
+    'January', 'February', 'March', 'April', 'May', 'June',
+    'July', 'August', 'September', 'October', 'November', 'December',
+  ];
+
+  return monthNames[month];
+}
+
+function padZero(timeUnit) {
+  return timeUnit < 10 ? `0${timeUnit}` : timeUnit;
```

Comment:
'magic number' 10 to constants

Reply:
https://github.com/SorokinTim/covid-dashboard-rss/pull/20/commits/1a94e2edd0f86e07fa1ad9c0f4356897fbf4bbd5